### DISCUSSION TITLE
Windows installer

### DISCUSSION THREAD

**Discussion by nickg** [nickg/nvc#876] [Announcements]
Starting from 1.12 there is now a standalone Windows MSI installer on the releases page:

[nvc-1.12.0.msi](https://github.com/nickg/nvc/releases/download/r1.12.0/nvc-1.12.0.msi)

This is built in the Clang MSYS2 environment but packages all the required dependencies so that no MSYS2 installation is required. This avoids the problem where dependencies of the MSYS2 packages could quickly become out-of-date (e.g. LLVM), and is also a bit simpler for the end user.

The installer will add the `nvc` binary to the system path so it should be immediately usable from command prompt or Powershell.

Please provide any feedback here.

**Comment by nickg** [comment]
You can also now install it with the Windows package manager:

```
winget install NickGasson.NVC
```

**Comment by ugrossm** [comment]
Compiling Xilinx libraries in Windows command prompt failed:
`o:\work>nvc --install ise`
`** Fatal: bash C:\ProgrammeUG\nvc\libexec\nvc\install-ise.sh failed with status -1`
Same result with Windows PowerShell.
When adding some `echo "Test"`  at the beginning of file `C:\ProgrammeUG\nvc\libexec\nvc\install-ise.sh` there is no additional output produced in the prompt.
The environment variable `%XILINX%` is set to `c:\Xilinx\14.7\ISE_DS\ISE`.
How to deal with those bash-Scripts in a standalone nvc installation on Windows (without MSYS2 installation)?
